src/pages/application/list/index.tsx

```diff
@@ -41,6 +42,10 @@ const ApplicationList: React.FC = () => {
 		dispatch(getApplications());
 	}, [state.updatedApplication, dispatch]);
 
+	useEffect(() => {
```

Comment:
I think in here you don't need three separate useEffects. You can write it in one useEffect. Put them in the array.

Reply:
No miyuru we used 3 useeffect for different use cases. We cant interagete those in one useEffect